AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_AutoComponent_NetworkInput.py

```diff
@@ -70,7 +70,7 @@ def find_unexpected_line(expected_line):
 
     level_name = "AutoComponent_NetworkInput"
     helper.init_idle()
-
+    general.set_cvar_integer('editorsv_port', 33452)
```

Comment:
Following up with test :)

Reply:
I thought we had a setup somewhere that would randomly pick a port to use each time the tests ran, which would keep collisions relatively low?

Reply:
editorsv_port is a relatively new cvar... maybe you're thinking of LY1.x?

Reply:
I'm unsure what this feature request would look like, from a test-specific standpoint. Can you clarify this ask, and what it would need to avoid? My best guess is that you would want another to use another higher-level context to control sending in client/server port information to the editor. This is already possible today, but would require organizing each test in a very specific way, which currently includes avoiding the EditorTest parallelization framework.

Automatically configuring/detecting valid client/server ports sounds like it may be a production feature of the networking system.

Reply:
Yeah I didn't think it was this cvar, but we have the ability to launch the Editor & AP running on different ports, we've had that since 1.x. It was the safest way to run multiple Asset Processors for different engine installs at the same time. I can't find it with a quick search in our tests, though. 

You can see the logic for setting the listening port in ReadListeningPortFromSettingsRegistry